x-pack/legacy/plugins/siem/index.ts

```diff
@@ -73,6 +73,7 @@ export function siem(kibana: any) {
       mappings: savedObjectMappings,
     },
     init(server: Server) {
+      server.injectUiAppVars('siem', async () => server.getInjectedUiAppVars('kibana'));
```

Comment:
For [reference](https://github.com/elastic/kibana/blob/master/test/plugin_functional/plugins/kbn_tp_visualize_embedding/index.js#L30-L37): 
```
// The following lines copy over some configuration variables from Kibana
// to this plugin. This will be needed when embedding visualizations, so that e.g.
// region map is able to get its configuration.
```